kagi/settings.py

```diff
@@ -0,0 +1,24 @@
+import os.path
+
+from django.conf import settings
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+RELYING_PARTY_ID = getattr(settings, "RELYING_PARTY_ID", "localhost")
+RELYING_PARTY_NAME = getattr(settings, "RELYING_PARTY_NAME", "Kagi Test Project")
+WEBAUTHN_ICON_URL = getattr(settings, "WEBAUTHN_ICON_URL", None)
+WEBAUTHN_TRUSTED_CERTIFICATES = getattr(
+    settings,
+    "WEBAUTHN_TRUSTED_CERTIFICATES",
+    os.path.join(BASE_DIR, "..", "trusted_attestation_roots"),
+)
+WEBAUTHN_TRUSTED_ATTESTATION_CERT_REQUIRED = getattr(
+    settings, "WEBAUTHN_TRUSTED_ATTESTATION_CERT_REQUIRED", True
```

Comment:
Based on the [related discussion about potential concerns with whitelisting attestation certs](https://github.com/solokeys/solo/issues/275#issuecomment-522170055), I get the impression it might be better to set the default value for this setting to `False`. What do you think?